examples/next/getting-started/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "next-headless-getting-started",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": ">=14.0.0"
```

Comment:
Can we also add the minimum npm version?

```
"engines": {
    "node": "14.x",
    "npm": "6.x"
  },
```

Reply:
@theodesp I saw you had [another issue in the backlog ](https://wpengine.atlassian.net/browse/FST-5) that was recommending `>=14.x` over `>=14.0.0`. Is that relevant here as well? Just want to make sure we're explicitly choosing one or the other. As far as I can tell they are equivalent 🤔

Reply:
Thats fine.